jobs/config.json

```diff
@@ -10343,6 +10343,26 @@
       "sig-cluster-lifecycle"
     ]
   },
+  "pull-kubernetes-e2e-kubeadm-gce-ipvs": {
+    "args": [
+      "--cluster=",
+      "--deployment=kubernetes-anywhere",
+      "--env-file=jobs/platform/gce.env",
+      "--gcp-zone=us-central1-f",
+      "--ginkgo-parallel=30",
+      "--kubeadm=pull",
+      "--kubernetes-anywhere-kubernetes-version=ci/latest",
+      "--kubernetes-anywhere-proxy-mode=ipvs"
```

Comment:
Is this feature `--kubernetes-anywhere-proxy-mode=ipvs` already available in kubernetes-anywhere?

Reply:
No, that pr still not approved. That's a problem.